### DISCUSSION TITLE
I Made Bruno Auto Test generator

### DISCUSSION THREAD

**Discussion by hi2rashid** [usebruno/bruno#1305] [Show and tell]
Hi, Part of our internal projects heavily using postman test scripts, migration seems head ache for few days.

I made a [Bruno Auto test generator](https://testsgenerator.net/)

Just feed a API response. It will generate test results based on the json body you gave.

You can generate multiple positive and negative test cases as well.

If i get good feedback, I have a plan to enhance it to autogenerate based on swagger openapi all API in one go.

happy 2024 :)

**Comment by hi2rashid** [comment]
<img width="896" alt="image" src="https://github.com/usebruno/bruno/assets/10250923/2bd12294-f0a4-4621-9a73-2063adbb7ba6">

<img width="724" alt="image" src="https://github.com/usebruno/bruno/assets/10250923/f03b8c42-7c4f-4088-8d2d-f76323aaf1ad">

**Reply by hxc-gxc** [reply]
Hi @hi2rashid, really nice work! Is there any way to look at the code base or the git repo please ?
